src/lib/permissions/index.ts

```diff
@@ -1,8 +1,20 @@
-export * from './interfaces';
-export { hasAccessToSpace } from './hasAccessToSpace';
-export * from './clients';
-export * from './forums';
-export * from './proposals';
-export * from './core';
-export * from './pages';
-export * from './spaces';
+// Space permission exports
+export * as SpacePermissionTypes from './spaces/interfaces';
+
+// Forum permission exports
+export * as ForumPermissionTypes from './forums/interfaces';
+export * from './forums/policies/index';
+
+// Proposal permission exports
+export * as ProposalPermissionTypes from './proposals/interfaces';
+
+// Page permission exports
+export * as PagePermissionTypes from './pages/interfaces';
+
+// Core and general permission exports
+export * as CorePermissionTypes from './core/interfaces';
+export { getPermissionAssignee } from './core/getPermissionAssignee';
+export * as PolicyTypes from './core/policies';
+
+export * as PermissionTypes from './interfaces';
+export { PermissionsApiClient } from './permissionsApiClient.class';
```

Comment:
I'm using index.ts here to declare what we actually export from the permissions folder (that does not have prisma dependency)

Should we skip having this index file, and just pull everything straight into src/lib/permissions.ts export?